astrokat/observe_main.py

```diff
@@ -122,13 +122,48 @@ def read_targets(target_items):
     return target_list
 
 
+def slew(session, target_info):
+    """Simple way to get telescope to slew to target
+
+    Parameters
+    ----------
+    session: `CaptureSession` object
+    target_info: dictionary with target observation info
+
+    """
+    katpt_tgt = target_info["target"]
+
+    user_logger.info("Slewing to first target")
```

Comment:
I agree, the naming of the function makes its purpose unclear, since the functionality only focus on the anticipated long slew to the first target.
Assumption warning:
In general moving from the last target of the previous observation to the first target of this observation will be the longest slew.

During this time AstroKAT does not capture data, it only start once on target, but then I do not want to run into the FBF wait of `capture_start`.

So I think I should fix both issues.
* I renamed the function to `initial_slew` to be explicit about the functionality.
* I also updated the info message since I like to target name feedback for the observer (thank you, good suggestion)